lib/modules/manager/devbox/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { Category } from '../../../constants';
+import { NixhubDatasource } from '../../datasource/nixhub';
+import { NodeVersionDatasource } from '../../datasource/node-version';
+
+export { extractPackageFile } from './extract';
+export { updateArtifacts } from './artifacts';
+
+export const supportsLockFileMaintenance = false; // not yet
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)devbox\\.json$'],
+  commitMessageTopic: 'devbox/{{depName}}',
```

Comment:
Are you sure you want commit messages and PR titles to look this way? e.g. "Update devbox/foo to v1.2.3". Normally we'd go with something like "Update foo to v1.2.3" or "Update foo crate to v1.2.3"